extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -536,4 +543,39 @@ export class AmpSlideScroll extends BaseSlides {
       event.stopPropagation();
     });
   }
+
+  /**
+   * @param {number} newSlideIndex
+   * @private
+   */
+  triggerAnalyticsEvent_(newSlideIndex) {
+    let direction = newSlideIndex - this.slideIndex_;
+    if (direction == 0) {
+      return;
+    } else if (Math.abs(direction) !== 1) {
+      direction = direction < 0 ? 1 : -1;
+    }
+    if (direction == 1) {
+      this.analyticsEvent_('amp-carousel-next');
+      this.analyticsEvent_('amp-carousel-change');
```

Comment:
Move `amp-carousel-change` above since you already exist for `direction == 0` at the top.

Also, any ideas for a name that is better than `change` for this?